apps/storefront/src/pages/order/orders.ts

```diff
@@ -0,0 +1,149 @@
+import B3Request from '@/shared/service/request/b3Fetch';
+import { OrderStatusItem } from '@/types';
+import { convertArrayToGraphql } from '@/utils';
+
+export type CustomerOrderNode = {
+  node: {
+    orderId?: string;
```

Comment:
surprised to see the id as optional
as well as `money` and `totalIncTax` 🤔

Reply:
don't know why, but they are setup as optional in the schema.

```
  money?: Maybe<Scalars['JSONString']['output']>;
  orderId?: Maybe<Scalars['String']['output']>;
  totalIncTax?: Maybe<Scalars['Float']['output']>;
```

we can remove the optionality in the new schema, but it is there in the current one 🤷